transcribe/scribe_modules/nvidia_smi.py

```diff
@@ -0,0 +1,27 @@
+from . import ScribeModuleBaseClass
+
+import sys
+
+
+from . lib.util import validate_length
+
+import json
+import xmltodict
+import pprint
```

Comment:
This was used for debugging, lets remove it.

Reply:
removed pprint